sections/appendix/uppaal.tex

```diff
@@ -0,0 +1,141 @@
+\section{Understanding a \uppaal model}\label{app:uppaal}
+
+The goal of this appendix is not to give an extensive guide on how to use \uppaal, but rather to serve as a short introduction on how to read the models described in this report.
+For a more in-depth description of how \uppaal works, we recommend the official tutorial for \uppaal 4.0 by Gerd Behrmann, Alexandre David and Kim G. Larsen \cite{uppaaltutorial}.
+
+\subsection{What is \uppaal?}
+\uppaal is an integrated tool environment for modeling and verification of timed automata.
+It is mainly used for simulating and verifying models of real-time systems \cite{uppaalintro} by means of property checking, where it can automatically generate a diagnostic trace to explain why a given property is, or is not, satisfied by the modeled system.\\
+This property checking is done using a reachability analysis, meaning that it will search through the state space of the system to find a state where a given property is true.
+Using property checking it is then possible to check for properties like whether the system can end up in a deadlock - or in the context of this system, it would be possible to check whether it is possible that one client is still in the lobby while the other clients are receiving in-game information such as positional data.
```

Comment:
```suggestion
Using property checking it is then possible to check for properties like whether or not the system can end up in a deadlock - or in the context of this system, it would be possible to check whether or not it is possible that one client is still in the lobby while the other clients are receiving in-game information such as positional data.
```